docs/templates.rst

```diff
@@ -24,6 +24,7 @@ The TemplateParam struct is defined as:
       Options      map[string]string
       Object       map[string]interface{}
       Phases       map[string]*Phase
+      PodOverride  map[string]interface{}
```

Comment:
```suggestion
      PodOverride  strategicpatch.JSONMap
```

Reply:
I thought that will confuse reader which format to use while passing the parameter from the Actionset. That's why I've kept it's type as `map[string]interface{}` in functions page also
https://github.com/kanisterio/kanister/pull/321/files#diff-610c3344a79db2ab08c4ef6b15842adeR416
Let me know that you think